File: bench/src/cpp/pi.cpp
#include <stdint.h>
int main(void){
  double pi=0.0, sign=1.0;
  for(uint64_t i=0;i<20000000ULL;i++){ uint64_t d=2*i+1; pi += sign/(double)d; sign=-sign; }
  double scaled=pi*4000000000.0; uint64_t check=(uint64_t)scaled; return (int)(check%256);
}

File: bench/src/ts/pi.ts
let pi: number = 0.0, sign: number = 1.0;
for(let i=0;i<20000000;i++){ const d=2*i+1; pi += sign/d; sign=-sign; }
const scaled: number = pi*4000000000.0;
process.exit(Math.floor(scaled) % 256);

export {};
